src/java.base/share/classes/java/time/zone/ZoneRules.java

```diff
@@ -830,7 +830,7 @@ public Duration getDaylightSavings(Instant instant) {
      *
      * @param instant  the instant to find the offset information for, not null, but null
```

Comment:
Hello Naoto, do you think this `@param` description too will have to be reworded? It seems to be a copy/paste of the `getDaylightSavings()` method.

Reply:
Hi Jai, looks like it was copied/pasted from `getStandardOffset()`, but you are right, the description needs to be corrected.